### PR TITLE
[FWK-317] change env var for queue names

### PR DESCRIPTION
**Pull request by aprakash-gc**

### CODE REVIEW

File: handlers/base.handler.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const log = require('../initializers/logger');
+const { log } = require('../initializers');
```
Comment (aprakash-gc) [inline handlers/base.handler.js:1]: i changed these imports to make sure that `dotenv` in the `initializers/index` runs, let me know if that's not right

File: handlers/consumer.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ class BaseConsumer {
 
   static async connect(options = {}) {
     try {
-      const queue = (process.env.APP_CLUSTER && process.env.APP_SERVICE) ? `${process.env.APP_CLUSTER}-${process.env.APP_SERVICE}-messages` : `local-${process.pid}-messages`;
+      const queue = (process.env.APP_CLUSTER && process.env.APP_CONSUMER_NAME) ? `${process.env.APP_CLUSTER}-${process.env.APP_CONSUMER_NAME}-messages` : `local-${process.pid}-messages`;
```
Comment (aprakash-gc) [inline handlers/consumer.js:63]: this is the real change for this ticket

**Comment by bearfact** [comment]
I have some concerns about changing the require statements / load order.  I just want to be sure this doesn't have any unintended consequences.

File: handlers/consumer.js (hunk 2 above)
Comment (bearfact) [reply handlers/consumer.js:63]: Late to the party but lets change this to be MESSAGE_BUS_CONSUMER_GROUP

**Comment by bearfact** [comment]
:+1:

**Pull request merged**